local_setup_guide.md

```diff
@@ -0,0 +1,146 @@
+# Setting Up Paired Development Environment
+
+## Front-end Setup
+
+Clone down the [front-end repo](https://github.com/DanielEFrampton/paired-fe):
```

Comment:
Would this still be to your github after this merge to setup the development environment?

Reply:
Whoops nope. Fix incoming.